### PR TITLE
chore: grant/details cleanup

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description 

- fixed alert to notify grant owner to switch to right metamask
  address.
- fix breaking UI on mobile + tablet on summary section
- disable change address button if address is same.
- added tooltip to give more info to grant owner can do only delete grant
- updated order of info displayed to ensure change grant owner gets pushed to bottom

##### Refers/Fixes

It should go in after https://github.com/gitcoinco/web/pull/4719
Just cause I built it on top of that PR

##### Testing

Check it out [HERE](https://embed.vidyard.com/share/ATvUHBNbMnqhDoKJWVMceF?)

UPDATE: [MADE A FEW MORE CHANGES TO MAKE UX LESS WEIRD ](https://embed.vidyard.com/share/KNPD7rRU1ynmBitDCoSzE8?)

### CODE REVIEW

File: app/grants/templates/grants/detail/info.html [hunk 1]
```diff
@@ -81,7 +81,7 @@ <h1 class="font-title-xl my-4 font-weight-bold">
 
       <div class="mb-2">
         <i style="width: 14px;" class="fab fa-ethereum mr-2 text-center"></i>
-        <a class="wallet-address" href="https://etherscan.io/address/{{ grant.admin_address }}"
+        <a id="wallet-address" class="wallet-address" href="https://etherscan.io/address/{{ grant.admin_address }}"
```
Comment (danlipert) [inline app/grants/templates/grants/detail/info.html]: Do we want to have a rinkeby option here (rinkeby.etherscan.io)

File: app/grants/templates/grants/detail/info.html [hunk 2]
```diff
@@ -180,9 +180,11 @@ <h1 class="font-title-xl my-4 font-weight-bold">
         <div class="font-body">
           <p class="sub-title mb-0">OWNER ADDRESS</p>
           <input id="contract_owner_address" class="p-0 m-0 inline-edit" name="contract_owner_address" value="{{ grant.contract_owner_address }}" disabled data-toggle="tooltip" data-placement="bottom" title="Beware! Please do not change this to an address you do NOT own.">
-          <button id="contract_owner_button" class="mt-2 button button--primary button--full hidden shadow-none font-weight-bold">
-            {% trans "Change Contract Owner" %}
-          </button>
+          <div id="contract_owner_button-container" tabindex="0" data-toggle="tooltip" data-placement="bottom" title="Grant owner address hasn't changed. Update the above field to enable this.">
+            <button id="contract_owner_button" data-toggle="tooltip" data-placement="bottom" title='Clicking this will trigger a metamsk confirmation' class="mt-2 disable-tooltip button button--primary button--full hidden shadow-none font-weight-bold">
```
Comment (danlipert) [inline app/grants/templates/grants/detail/info.html]: Typo here `metamsk`

File: app/grants/templates/grants/detail/info.html (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/templates/grants/detail/info.html]: Ah i shalll fix that up ^_^

**Comment by thelostone-mc** [comment]
@danlipert addressed your comments and throw in a few more changes to make UX less weird 
Check [here](https://embed.vidyard.com/share/KNPD7rRU1ynmBitDCoSzE8?)

File: app/grants/templates/grants/detail/info.html [hunk 3]
```diff
@@ -81,7 +81,7 @@ <h1 class="font-title-xl my-4 font-weight-bold">
 
       <div class="mb-2">
         <i style="width: 14px;" class="fab fa-ethereum mr-2 text-center"></i>
-        <a id="wallet-address" class="wallet-address" href="https://etherscan.io/address/{{ grant.admin_address }}"
+        <a id="wallet-address" class="wallet-address"
```
Comment (danlipert) [inline app/grants/templates/grants/detail/info.html]: Should we use an anchor tag for this? https://www.w3.org/TR/html5/text-level-semantics.html#the-a-element

Comment (thelostone-mc) [reply app/grants/templates/grants/detail/info.html]: It is an anchor tag ! The href gets populated [here](https://github.com/gitcoinco/web/pull/4720/files#diff-ce752a635ca0517ae9c9403aadbd5f09R94)

This ensures that if the grant gets created on mainnet -> the main-net etherscan link shows up 
. If it's rinkeby -> the rinkeby one shows up 

I can leave an empty href tag if that helps @danlipert

**Pull request merged**